terra_notebook_utils/cli/__init__.py

```diff
@@ -23,6 +25,15 @@ def load(cls):
     def write(cls):
         with open(cls.path, "w") as fh:
             fh.write(json.dumps(cls.info, indent=2))
+
+    @classmethod
+    def resolve(cls, default_workspace: str, default_namespace: str):
+        workspace = default_workspace or (cls.info['workspace'] or WORKSPACE_NAME)
+        namespace = default_namespace or (cls.info['workspace_google_project'] or WORKSPACE_GOOGLE_PROJECT)
```

Comment:
I'd expect a default to be the last item resolved in a short circuit.  This looks like it will always choose the default if it exists, is that correct?

```
>>> i = '' or None or 'Best Choice' or 'default'
>>> print(i)
Best Choice
```

Reply:
I'll change the names away from "default_*"